lib/postgres_v3_experimental.dart

```diff
@@ -127,11 +127,16 @@ abstract class PgSession {
   /// optimization can be applied also depends on the parameters chosen, so
   /// there is no guarantee that the [PgResult] from a [ignoreRows] excution has
   /// no rows.
+  /// 
+  /// When [useSimpleQueryProtocol] is set to true, the implementation will use
+  /// the Simple Query Protocol. Please note, a query with [parameters] cannot
+  /// be used with this protocol. 
   Future<PgResult> execute(
     Object /* String | PgSql */ query, {
     Object? /* List<Object?|PgTypedParameter> | Map<String, Object?|PgTypedParameter> */
         parameters,
     bool ignoreRows = false,
```

Comment:
Since we'll receive the rows anyway and just chose to ignore them, perhaps the `ignoreRows` parameter doesn't make much sense anymore when we have a `useSimpleQueryProtocol` parameter as well.

So maybe the `ignoreRows` parameter should just be removed in favor of `useSimpleQueryProtocol`?

Reply:
I wasn't sure what is the use of `ignoreRows` so I left it as is.. I will take a closer look later today and update the PR accordingly.

I will also expand on the docs for `useSimpleQueryProtocol` and mention some of its use cases.

Reply:
If possible, I would like to keep `ignoreRows`, because if the caller needs only a non-error execution, we can spare a bit of processing. I don't mind having `useSimpleQueryProtocol` but only if that's the only way to do it (see my main comment about having it at connection level or with a callback).